typescript/infra/scripts/warp-routes/monitor-warp-routes-balances.ts

```diff
@@ -55,10 +55,11 @@ const xERC20LimitsGauge = new Gauge({
   name: 'hyperlane_xerc20_limits',
   help: 'Current minting and burning limits of xERC20 tokens',
   registers: [metricsRegister],
-  labelNames: ['chain_name', 'limit_type'],
+  labelNames: ['chain_name', 'limit_type', 'token_name'],
```

Comment:
Adding token name label so that we can distinguish renzo warp route limits in the dashboard